roles/grafana/tasks/grafana.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: install grafana
+  yum: name={{ item }} state=present
```

Comment:
As in roles/collectd/tasks/main.yml  use new-style yaml syntax 
yum:
  name: {{ item }}
  state: present